### PR TITLE
doc: improve several explanations

### LINKED ISSUE #1138: doc: Improve several explanations
Improve documentation related to:

- [ ] WS(S) ports in Security.
- [ ] E.164 number text fields: escape + hint.
- [ ] Importing several price CSV in the same destination-prices row.
- [ ] Difference between partial holiday event and timetable events.

### PR DESCRIPTION
**Pull request by cruzccl**
#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [x] Changes have been tested locally
- [x] Fixes an existing issue (Fixes #1138) 

#### Description

- WS(S) ports in security
- E.164 number in regexp must escape plus symbol.
- Hints about importing multiple price CSV in the same destination-rates row.
- Difference between partial holiday event and timetible events.

### CODE REVIEW

**Comment by Kaian** [comment]
Use Pull request templates :angry:

File: doc/sphinx/administration_portal/client/vpbx/routing_tools/match_lists.rst [hunk 1]
```diff
@@ -27,4 +27,7 @@ The section **Client configuration** > **Match Lists** allows to configure
 different items that will group the numbers and patterns.
 
 As shown in **List of Match List Patterns**, a match list can contain specific numbers or groups using
-`Regular Expressions <http://php.net/manual/en/reference.pcre.pattern.syntax.php>`_
\ No newline at end of file
+`Regular Expressions <http://php.net/manual/en/reference.pcre.pattern.syntax.php>`_
+
+.. error:: **Regular expressions** of Match List patterns must be **in E.164** format and **plus symbol must be
+               escaped** (e.g. ^\\+34 for all spanish numbers).
```
Comment (Kaian) [inline doc/sphinx/administration_portal/client/vpbx/routing_tools/match_lists.rst]: `^\+34` is the correct format for the spanish regular expression

File: doc/sphinx/administration_portal/client/vpbx/user_configuration/call_acls.rst [hunk 2]
```diff
@@ -33,7 +33,9 @@ the call permission.
 For more information of how MatchLists patterns are created, please refer to section
 :ref:`match_lists`.
 
-.. attention:: **Regular expressions of Match List patterns must be in E.164 format**.
+.. error:: **Regular expressions** of Match List patterns must be **in E.164** format and **plus symbol must be
+               escaped** (e.g. ^\\+34 for all spanish numbers).
```
Comment (Kaian) [inline doc/sphinx/administration_portal/client/vpbx/user_configuration/call_acls.rst]: `^\+34` is the correct format for the spanish regular expression

File: doc/sphinx/locale/es/LC_MESSAGES/administration_portal.po [hunk 3]
```diff
@@ -8406,3 +8406,9 @@ msgstr ""
 #~ msgid "Download a CSV with each rating plan."
 #~ msgstr ""
 
+#~ msgid "No recording enable/disable feature: all calls will be recorded."
+#~ msgstr ""
+
+#~ msgid "**Regular expressions of Match List patterns must be in E.164 format**."
+#~ msgstr ""
+
```
Comment (Kaian) [inline doc/sphinx/locale/es/LC_MESSAGES/administration_portal.po]: Remove old strings no longer used

File: doc/sphinx/locale/es/LC_MESSAGES/security_and_maintenance.po [hunk 4]
```diff
@@ -597,45 +597,60 @@ msgstr ""
...
 
 #: ../../security_and_maintenance/security/firewall.rst:31
+msgid "Port 10081 (TCP) for Websocket secure connections (WSS)."
+msgstr ""
+
+#: ../../security_and_maintenance/security/firewall.rst:33
+#, fuzzy
+msgid "**Provider side SIP signalling**:"
+msgstr "**Señalización SIP**:"
+
```
Comment (Kaian) [inline doc/sphinx/locale/es/LC_MESSAGES/security_and_maintenance.po]: Remove fuzzy tag and leave translation empty

File: doc/sphinx/locale/es/LC_MESSAGES/security_and_maintenance.po [hunk 5]
```diff
@@ -597,45 +597,60 @@ msgstr ""
 "funcionar correctamente:"
 
 #: ../../security_and_maintenance/security/firewall.rst:23
-msgid "**SIP signalling**:"
+#, fuzzy
+msgid "**Client side SIP signalling**:"
 msgstr "**Señalización SIP**:"
```
Comment (Kaian) [inline doc/sphinx/locale/es/LC_MESSAGES/security_and_maintenance.po]: Remove fuzzy tag and leave translation empty

File: doc/sphinx/locale/es/LC_MESSAGES/security_and_maintenance.po [hunk 6]
```diff
@@ -663,3 +678,12 @@ msgstr ""
...
 
+#~ msgid ""
+#~ "Port 7060 (TCP/UDP) y 7061 TCP "
+#~ "(just in case both ProxyUsers and "
+#~ "ProxyTrunks share IP)"
+#~ msgstr ""
+#~ "Puerto 7060 (TCP/UDP) y 7061 TCP "
+#~ "(si KamUsers y KamTrunks comparten un"
+#~ " única IP pública)"
+
```
Comment (Kaian) [inline doc/sphinx/locale/es/LC_MESSAGES/security_and_maintenance.po]: Remove old strings no longer used

File: doc/sphinx/administration_portal/client/vpbx/routing_tools/match_lists.rst (hunk 1 above)
Comment (cruzccl) [reply doc/sphinx/administration_portal/client/vpbx/routing_tools/match_lists.rst]: Two slashes are needed to show just one ;)

File: doc/sphinx/administration_portal/client/vpbx/user_configuration/call_acls.rst (hunk 2 above)
Comment (cruzccl) [reply doc/sphinx/administration_portal/client/vpbx/user_configuration/call_acls.rst]: Two slashes are needed to show just one ;)

**Pull request merged**